front/src/app/components/forms/convention/VerificationActionButton.tsx

```diff
@@ -121,6 +153,11 @@ export const VerificationActionButton = ({
   };
 
   const conventionId = convention.id;
+  const onClick = () =>
```

Comment:
```suggestion
  const onActionButtonClick = () =>
```

Ou (si onActionButtonClick ne convient pas) quelque chose qui exprime ce sur quoi on click